README.md

```diff
@@ -13,8 +13,10 @@ that they are not required to be valid UTF-8.
 
 https://docs.rs/bstr
 
-The documentation includes a section on why you might want to use this crate.
+### When should I use byte string
```

Comment:
Sorry for the churn, but my previous comment was meant to say that this heading should match what's in the docs, which is "When should I use byte strings?" Could you change it to that please?